docsteady/__init__.py

```diff
@@ -353,8 +353,11 @@ def baseline_ve(format, username, password, details, component, subcomponent, pa
         autoescape=None
     )
 
+    if Config.MODE_PREFIX != "dm-":
```

Comment:
I am fine with your proposal, but os.path.exists requires probably an absolute path.

I would rather keep my original implementation. Note that Config.MODE_PREFIX is set in the cli method above, so I don't understand why this implementation is awkward. Should maybe use a different variable instead of setting to Config.MODE_PREFIX = ""?